checkstyle-tester/diff.groovy

```diff
@@ -257,7 +256,6 @@ def generateCheckstyleReport(cfg) {
 
     def checkstyleConfig = cfg.checkstyleCfg
     def checkstyleVersion = cfg.checkstyleVersion
-    def failsOnError = cfg.failsOnError
```

Comment:
There is no CLI option `failsOnError`, no documentation, and no usages. Main repo passes it as an extra maven option.